src/include/common/types/types.h

```diff
@@ -87,6 +90,7 @@ class DataType {
 public:
     DataTypeID typeID;
     std::unique_ptr<DataType> childType;
```

Comment:
Add a comment here above `    std::unique_ptr<DataType> childType;`: // Below are for the LIST data type.

Reply:
added comment